api/server/models/user.js

```diff
@@ -324,4 +324,53 @@ module.exports = function (User) {
     );
 //********************************* END RESET PASSWORD **********************************
 
+//********************************* START LIST OF ALL USERS **********************************
+    User.list_users = function (req, page, currentTimestamp, callback) {
+        if (typeof req.accessToken == 'undefined' || req.accessToken == null || req.accessToken == '' || typeof req.accessToken.userId == 'undefined' || req.accessToken.userId == '') {
+            callback(null, 0, 'UnAuthorized', {});
+        } else {
+            var num = 0;
+            var access_token_userid = req.accessToken.userId;
+            if (typeof page != 'undefined') {
+                num = page;
+            }
+            User.findById(access_token_userid, function (err, user) {
+                if (err) {
+                    callback(null, 0, err);
+                } else {
+                    User.find({
+                        order: 'last_seen DESC',
+                        skip: num * 10,
+                        limit: 10
+                    }, function (err, result) {
+                        if (err) {
+                            callback(null, 0, err);
+                        }
+                        else {
+                            if (result.length > 0)
+                                callback(null, 1, result);
```

Comment:
dont return entire user table. only data which is required in frontend 
ask yuvraj what all he needs in frontend